Libraries/Opc.Ua.Client/DefaultActivitySessionFactory.cs

```diff
@@ -0,0 +1,148 @@
+/* ========================================================================
+ * Copyright (c) 2005-2023 The OPC Foundation, Inc. All rights reserved.
```

Comment:
Better file/class name might be `SessionActivitySourceFactory`

Reply:
agreed and updated